locales/de/subtitles/lucien-ducastel/lucien-ducastel-01.yml

```diff
@@ -19,13 +19,13 @@ de:
           s: 3
           from: 00:00:12,25
           to: 00:00:25,18
-          text: For three years, I was an apprentice butcher, but did not like it
-            at all. I started working in construction, as a metal worker.
+          text: Ich war drei Jahre lang Metzgerlehrling, aber das gefiel mir gar
+            nicht. Dann fing ich an, auf dem Bau zu arbeiten, im Metallbau.
```

Comment:
The comma before "aber" is mandatory. 
I'm a fan of commas :-D so I generally tend to put a comma before an infinitive with "zu". But that is optional, you're right. So we can delete the comma after "an" if you prefer it :)